server/repository/src/mock/period_and_period_schedule.rs

```diff
@@ -0,0 +1,29 @@
+use chrono::NaiveDate;
+
+use crate::PeriodRow;
+use crate::PeriodScheduleRow;
+
+pub fn mock_period_schedule_1() -> PeriodScheduleRow {
+    PeriodScheduleRow {
+        id: "mock_period_schedule_1".to_string(),
+        name: "Bi Weekly".to_string(),
```

Comment:
nit:
```suggestion
        name: "Monthly".to_string(),
```
Just because the associated period is for a month...
Not a major :)